apis/v1beta1/httproute_types.go

```diff
@@ -631,6 +639,14 @@ const (
 	// Support in HTTPBackendRef: Extended
 	HTTPRouteFilterRequestHeaderModifier HTTPRouteFilterType = "RequestHeaderModifier"
 
+	// HTTPRouteFilterResponseHeaderModifier can be used to add or remove an HTTP
+	// header from an HTTP response before it is sent to the client.
+	//
+	// Support in HTTPRouteRule: Extended
+	//
+	// Support in HTTPBackendRef: Extended
+	HTTPRouteFilterResponseHeaderModifier HTTPRouteFilterType = "ResponseHeaderModifier"
```

Comment:
We should mark this value as experimental like this:

https://github.com/kubernetes-sigs/gateway-api/blob/2e82b50d2b91cc2f9c4dc96181fdcc9131c27482/apis/v1beta1/shared_types.go#L113